### PR TITLE
Added required individual components of VS to readme

### PR DESCRIPTION
**Pull request by konradsikorski**
**Related Issues**
Items related issues by this PR.
- Related #8467

**Details of Changes**
Update readme with all required Individual components to install with VIsual studio to be able to build this project

**Validation**
How did you test these changes?
- [x] Previewed as a markdown file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,9 +30,12 @@ Have any questions? Check out our [documentation site](https://files.community/d
 
 ### 1: Prerequisites
 
+- Windows 11
 - [Git](https://git-scm.com)
-- [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) with the UWP Development Kit, including C++ UWP build tools.
-- C++ ATL for UWP build tools
+- [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) with the UWP Development Kit and following individual components:
+    - Windows 11 SDK (10.0.22000)
```
Comment (yaichenbaum) [inline README.md:36]: Windows 10 works as well
```suggestion
```

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

**Comment by yaichenbaum** [comment]
Can you update the website documentation as wel?

File: README.md [hunk 2]
```diff
@@ -30,9 +30,12 @@ Have any questions? Check out our [documentation site](https://files.community/d
 
 ### 1: Prerequisites
 
+- Windows 11
```
Comment (yaichenbaum) [inline README.md:33]: Windows 10 is also supported

Comment (konradsikorski) [reply README.md]: You are right, i have removed this point

**Review by yaichenbaum — APPROVED** [review]

**Pull request merged**